test/system/comment_test.rb

```diff
@@ -447,6 +447,37 @@ def get_path(page_type, path)
       assert_selector('.btn[data-original-title="Help"]', count: 1)
     end
 
+    test "#{page_type_string}: IMMEDIATE rich-text input works in MAIN form" do
+      visit get_path(page_type, nodes(node_name).path)
+      main_comment_form =  page.find('h4', text: /Post comment|Post Comment/).find(:xpath, '..') # title text on wikis is 'Post comment'
+      main_comment_form.find("[data-original-title='Bold']").click
+      text_input_value = main_comment_form.find('#text-input').value
+      assert_equal(text_input_value, '****')
+    end
+
```

Comment:
Two new tests, these were the most likely scenarios for screw-ups when I did my manual testing.